Shared/Models/AnimationCycle.swift

```diff
@@ -14,4 +14,5 @@ struct AnimationCycle {
     var shouldConcatenate: Bool
     var scrollToIndexPath: IndexPath?
     var scrollPosition: UICollectionView.ScrollPosition = .centeredHorizontally
+    var scrollToOffset: CGPoint? = nil
```

Comment:
The animation cycle can now take a specific offset to scroll to. The time machine layout doesn't work when attempting to scroll to a specific index path.